src/chrome/content/rules/AOL.xml

```diff
@@ -190,12 +184,6 @@
 
 	<target host="expapi.oscar.aol.com" />
 
-		<!--	Added by AOL, required for http logout.
-								-->
-		<exclusion pattern="^http://my\.screenname\.aol\.com/_cqr/logout/" />
-
-			<test url="http://my.screenname.aol.com/_cqr/logout/" />
-
```

Comment:
As I understand it, the specified exclusion pattern defines just the root of URLs to be excluded.  If you do an HTTP Trace on a logout of www.aol.com, it will actually invoke http://my.screenname.aol.com/_cqr/logout/mcLogout.psp with some query parameters.  I don't work on the application at AOL anymore, but I did confirm with the developers that this is still needed.  Thanks!